packages/camera/camera/ios/Classes/CameraPlugin_Test.h

```diff
@@ -5,14 +5,18 @@
 // This header is available in the Test module. Import via "@import camera.Test;"
 
 #import <camera/CameraPlugin.h>
+#import <camera/FLTCam.h>
 #import <camera/FLTThreadSafeFlutterResult.h>
 
 /// Methods exposed for unit testing.
 @interface CameraPlugin ()
 
-// All FLTCam's state access and capture session related operations should be on run on this queue.
+/// All FLTCam's state access and capture session related operations should be on run on this queue.
 @property(nonatomic, strong) dispatch_queue_t captureSessionQueue;
 
+/// An internal camera object that manages camera's state and performs camera operations.
+@property(nonatomic, strong) FLTCam *camera;
```

Comment:
I don't love that this is exposed as readwrite now, but I can't think of a better way to do it since a test-only convenience initializer isn't great either since `FLTCam` is created later.

Reply:
Oh this was supposed to be `readwrite` in the first place. The old `readonly` worked before only because we used the ivar and bypassed the attribute's restriction: 
```
      _camera = cam;
```

Reply:
`readwrite` in the implementation though, not to, say, consumers of the plugin (who can also import the test module).  But it's fine I don't have a better idea for testing this.

Reply:
yeah should be fine. we already have many internal APIs exposed under this test module (eg `CameraPlugin_Test.h`). it's very unlikely for non-test code to import it.